### ISSUE TITLE
Nuget

### ISSUE THREAD

**Issue by viiny123** [SilverCard/DanfeSharp#18]
Bom dia, brother quando baixei o projeto deu problema de referencia do PDFClown, então adicionei ele via nuget, https://www.nuget.org/packages/PDFClown.NET/, rodei os testes e passaram normalmente, acredito que agora seja possível vc adicionar o mesmo no nuget.

**Comment by felixsevero** [comment]
Não conheço muito o projeto do PDF Clown, mas caso o referenciado no Nuget seja o mesmo utilizado por este projeto, seria ótimo estar disponível no Nuget. Gostaria da opinião do @SilverCard sobre isso, pois o DanfeSharp é muito importante para muitos desenvolvedores.

**Comment by DSBRBrasil** [comment]
Será que o nobre autor abandonou o projeto ? Caso afirmativo,é uma pena, por que esse projeto é muito bom. Estou trabalhando num emissor de NFe e NFce gratuito, entao estou a procura de DANFE para o projeto

**Comment by scalarerp** [comment]
> Será que o nobre autor abandonou o projeto ? Caso afirmativo,é uma pena, por que esse projeto é muito bom. Estou trabalhando num emissor de NFe e NFce gratuito, entao estou a procura de DANFE para o projeto

Tenho o projeto todo rodando em .netcore2.0 e funcionando com o Zeus.Dfe também em .netcore2.0, mas não tenho parcerias para dar continuidade em nenhum dos dois projetos, também não tenho tempo para emcabeçar isso, se alguém  estiver disponivel para começar eu topo.

**Comment by felixsevero** [comment]
@scalarerp Se você fez modificações nesses projetos, pode simplesmente dar um Pull Request, que a comunidade vai dar continuidade.

**Comment by SilverCard** [comment]
Eu não abandonei o projeto, caso encontre algum problema ou precise implementar algo descrito numa nota técnica, por favor, abra um issue aqui no GitHub.

**Comment by scalarerp** [comment]
> Eu não abandonei o projeto, caso encontre algum problema ou precise implementar algo descrito numa nota técnica, por favor, abra um issue aqui no GitHub.

Parabéns, não abandone, é um legado que deve te orgulhar pois ajudou muita gente.  Acho só que deve seguir em frente e implementar o .net standart 2.0, o restante é show de bola.

**Comment by DSBRBrasil** [comment]
> Eu não abandonei o projeto, caso encontre algum problema ou precise implementar algo descrito numa nota técnica, por favor, abra um issue aqui no GitHub.

Maravilha !!! Então.... estou terminando um emissor de NFe e NFCe gratuito em net. standard 2 e vou disponibilizar aqui no Git, agora, estou precisando de DANFes que possam ser retornados em array de bytes ...
Será possível nós começarmos essa parceria?

**Comment by DSBRBrasil** [comment]
> > Será que o nobre autor abandonou o projeto ? Caso afirmativo,é uma pena, por que esse projeto é muito bom. Estou trabalhando num emissor de NFe e NFce gratuito, entao estou a procura de DANFE para o projeto
> 
> Tenho o projeto todo rodando em .netcore2.0 e funcionando com o Zeus.Dfe também em .netcore2.0, mas não tenho parcerias para dar continuidade em nenhum dos dois projetos, também não tenho tempo para emcabeçar isso, se alguém estiver disponivel para começar eu topo.

A ideia é correr do Zeus, (faço uso dele também), você já olhou o Zeus por dentro? Tem certeza que ainda vai continuar? Já viu oque corre corre quando se muda algum endereço web ? Já teve problemas com enums?

**Comment by felixsevero** [comment]
> 
> 
> > > Será que o nobre autor abandonou o projeto ? Caso afirmativo,é uma pena, por que esse projeto é muito bom. Estou trabalhando num emissor de NFe e NFce gratuito, entao estou a procura de DANFE para o projeto
> > 
> > 
> > Tenho o projeto todo rodando em .netcore2.0 e funcionando com o Zeus.Dfe também em .netcore2.0, mas não tenho parcerias para dar continuidade em nenhum dos dois projetos, também não tenho tempo para emcabeçar isso, se alguém estiver disponivel para começar eu topo.
> 
> A ideia é correr do Zeus, (faço uso dele também), você já olhou o Zeus por dentro? Tem certeza que ainda vai continuar? Já viu oque corre corre quando se muda algum endereço web ? Já teve problemas com enums?

Cara, eu utilizo o Zeus e ele me atende perfeitamente. Não creio que ele seja um problema. Ele parece ser bem construído e estar evoluindo bem.

**Comment by DSBRBrasil** [comment]
> > > > Será que o nobre autor abandonou o projeto ? Caso afirmativo,é uma pena, por que esse projeto é muito bom. Estou trabalhando num emissor de NFe e NFce gratuito, entao estou a procura de DANFE para o projeto
> > > 
> > > 
> > > Tenho o projeto todo rodando em .netcore2.0 e funcionando com o Zeus.Dfe também em .netcore2.0, mas não tenho parcerias para dar continuidade em nenhum dos dois projetos, também não tenho tempo para emcabeçar isso, se alguém estiver disponivel para começar eu topo.
> > 
> > 
> > A ideia é correr do Zeus, (faço uso dele também), você já olhou o Zeus por dentro? Tem certeza que ainda vai continuar? Já viu oque corre corre quando se muda algum endereço web ? Já teve problemas com enums?
> 
> Cara, eu utilizo o Zeus e ele me atende perfeitamente. Não creio que ele seja um problema. Ele parece ser bem construído e estar evoluindo bem.

Na época ele foi bem construído sim, o problema foi que ao longo do tempo ele foi sendo modificado e terminou sendo "marretado".  Você já teve a oportunidade de analisar o código por dentro? O Zeus me atende bem também, mas, depois que vi o código por dentro e a responsabilidade da modificação do projeto centrado em uma única pessoal eu concluir ser uma bolha que um dia vai estourar e deixar muita gente na mão. Mas, são as minhas conclusões, fique tranquilo, o Zeus da rodando bem.

**Comment by felixsevero** [comment]
> 
> 
> > > > > Será que o nobre autor abandonou o projeto ? Caso afirmativo,é uma pena, por que esse projeto é muito bom. Estou trabalhando num emissor de NFe e NFce gratuito, entao estou a procura de DANFE para o projeto
> > > > 
> > > > 
> > > > Tenho o projeto todo rodando em .netcore2.0 e funcionando com o Zeus.Dfe também em .netcore2.0, mas não tenho parcerias para dar continuidade em nenhum dos dois projetos, também não tenho tempo para emcabeçar isso, se alguém estiver disponivel para começar eu topo.
> > > 
> > > 
> > > A ideia é correr do Zeus, (faço uso dele também), você já olhou o Zeus por dentro? Tem certeza que ainda vai continuar? Já viu oque corre corre quando se muda algum endereço web ? Já teve problemas com enums?
> > 
> > 
> > Cara, eu utilizo o Zeus e ele me atende perfeitamente. Não creio que ele seja um problema. Ele parece ser bem construído e estar evoluindo bem.
> 
> Na época ele foi bem construído sim, o problema foi que ao longo do tempo ele foi sendo modificado e terminou sendo "marretado". Você já teve a oportunidade de analisar o código por dentro? O Zeus me atende bem também, mas, depois que vi o código por dentro e a responsabilidade da modificação do projeto centrado em uma única pessoal eu concluir ser uma bolha que um dia vai estourar e deixar muita gente na mão. Mas, são as minhas conclusões, fique tranquilo, o Zeus da rodando bem.

Sim, ele tem alguns códigos ruins, mas possui alguns membros que cooperam muito pra manter ele funcional, e acredito que ele está sendo refatorado aos poucos. Acho que a tendência é ele ir melhorando com o tempo, ao invés do oposto, até porque é um projeto Open Source, o que ajuda muito. Em último caso, poderia se fazer um fork dele, mas não acho que será necessário. Nós também podemos debater e contribuir com o que acharmos necessário, isso é algo benéfico para todos.
